source/cloud-security/gcp/prerequisites/pubsub.rst

```diff
@@ -26,6 +34,10 @@ Every publishing application sends messages to topics. Wazuh will retrieve the l
 Create a subscription
```

Comment:
I would say "custom subscription" for it is only an option when not using the default one